src/sentry/workflow_engine/migration_helpers/utils.py

```diff
@@ -0,0 +1,69 @@
+from typing import cast
+
+from sentry.incidents.models.alert_rule import AlertRule, AlertRuleTrigger, AlertRuleTriggerAction
+from sentry.models.team import Team
+from sentry.users.services.user import RpcUser
+
+MAX_CHARS = 248  # (256 minus space for '...(+3_)')
+
+
+def get_action_description(action: AlertRuleTriggerAction) -> str:
```

Comment:
I based this off of https://github.com/getsentry/sentry/blob/master/src/sentry/incidents/endpoints/serializers/alert_rule_trigger_action.py#L9-L9 didn't seem worth it to refactor to use both since a) we'll be getting rid of the serializer and b) I changed all the text

Reply:
Some of these read a little strangely like "PagerDuty to #my-team" but I was shortening it from `"Send a {priority} PagerDuty notification to {target_display}"`, very open to other ideas for these! "Slack #michelle" rolls off the tongue a lot nicer than some of these